assets/airports/eddh.json

```diff
@@ -1,844 +1,1435 @@
+                "33": ["DH250", "DH253"]
+            },
+            "body": ["LUB"],
+            "exitPoints": {
+                "LUB": []
+            },
+            "draw": [
+                ["HOS", "LUB*"],
+                ["GT", "DH104", "LUB"],
+                ["HAM", "LUB"]
+            ]
+        },
+        "RAMAR": {
+            "icao": "RAMAR",
+            "name": "Ramar",
+            "suffix": {"05":"3C", "15":"3D", "23":"3B", "33":"3G"},
+            "rwy": {
+                "05": ["HOS", "DH154"],
+                "15": [["^DH208", "A23+"], "DH209", "DH212"],
+                "23": ["^DH111", "DH102", "GT", "DH104", "DH110"],
+                "33": ["DH250", "DH253", "DH259"]
+            },
+            "body": ["LYE", "RAMAR"],
+            "exitPoints": {
+                "RAMAR": []
+            },
+            "draw": [
+                ["HOS", "DH154", "LYE"],
+                ["DH110", "LYE"],
+                ["DH253", "DH259"],
+                ["DH202", "DH208", "DH209", "DH212", "LYE", "RAMAR*"]
+            ]
+        },
+        "WSN": {
+            "icao": "WSN",
+            "name": "Weser",
+            "suffix": {"05":"2C", "15":"1D", "23":"1B", "33":"2G"},
             "rwy": {
                 "05": ["DH150", "DH153", "LBV"],
-                "23": ["DH105", "DH117"],
-                "15": [["DH202", "A20+"], "DH216", "DH221"],
+                "15": [["^DH202", "A20+"], "DH216"],
+                "23": ["DH100"],
                 "33": ["DH254", "LBV"]
             },
-            "body": [],
+            "body": ["WSN"],
             "exitPoints": {
-                "BASUM": ["BASUM"]
+                "WSN": []
             },
-            "draw": [["DH105", "DH117"], ["LBV", "BASUM"]]
+            "draw": [
+                ["LBV", "WSN*"],
+                ["DH100", "WSN"],
+                ["DH250", "DH254", "LBV"],
+                ["DH216", "WSN"]
+            ]
         }
     },
     "stars": {
         "BOGMU": {
             "icao": "BOGMU",
             "name": "Bogmu",
+            "suffix": {"05":"05", "15":"15", "23":"23", "33":"33"},
             "entryPoints": {
-                "N850": ["MIC", ["NUSTA", "A170"], ["NIGAD", "A130"]]
+                "BOGMU": []
             },
-            "body": [["BOGMU", "A100|S250"]],
+            "body": ["BOGMU"],
             "rwy": {
-                "05": [["DH225", "S220"], "DH230", "DH235", "DH255", "DH256", "DH257", "DH025", "ANEXI"],
-                "15": [["DH412", "S220"], "DH416", "DH455", "DH456", "DH457", "DH458"],
-                "23": [["DH634", "S220"], "DH635", "DH655", "DH656", "DH657", "DH658"],
-                "33": [["DH829", "S220"], "DH830", "DH835", "DH854", "DH855", "DH856", "DH857", "DH858"]
+                "05": [["DH225", "S220"], "DH226", "DH227", "DH228", "DH229", ["DH230", "S220"], "DH231", "DH232", "DH233", "DH234", ["DH235", "S220"], "DH255", "DH256", "DH257", "DH025", ["ANEXI", "S170+"]],
```

Comment:
- Where you have `DH025` my charts show `DH258`. They appear to have the same location.
- `ANEXI` appears to be `A30+|S170+`.